src/pages/OrganizerAddModal.tsx

```diff
@@ -102,8 +105,8 @@ const OrganizerAddModal = ({
 
   const urlRegisterProps = register('url');
 
-  const watchedUrl = watch('url');
-  const watchedCountry = watch('address.country');
+  const watchedUrl = useWatch({ control, name: 'url' });
+  const watchedCountry = useWatch({ control, name: 'address.country' });
```

Comment:
Is it possible to write it like this?
```suggestion
  const [watchedUrl, watchedCountry] = useWatch({ control, name: ['url', 'address.country'] });
```

Reply:
Yep should be possible, I'll change it